CMakeLists.txt

```diff
@@ -209,11 +222,19 @@ endif()
 #else()
     add_library(SDL2main STATIC src/SDLmain/dummy/SDL_dummy_main.c)
 #endif()
-#add_library(SDL2::SDL2ain ALIAS SDL2main)
+add_library(SDL2::SDL2main ALIAS SDL2main)
 target_include_directories(SDL2main PRIVATE "include/SDL2")
+if(MINGW OR CYGWIN)
+  cmake_minimum_required(VERSION 3.13)
```

Comment:
Do we have a better (another) way for this than requiring cmake >= 3.13?

Reply:
Oh, this can be removed. It was `target_link_options` before, as done in SDL itself.

Reply:
SDL(2) itself has the same thing, sadly.

Reply:
I think we can get away with `target_link_libraries`.
On the other hand, nobody complained that it didn't work for their older CMake.
So everybody was able to upgrade.

Reply:
> On the other hand, nobody complained that it didn't work for their older CMake.

I did :))) I'm still with 3.9.6 on my everyday machine, can't build a newer one.

Reply:
How do you build SDL_image with CMake then?

Reply:
> How do you build SDL_image with CMake then?

I don't, I just use autotools.

Reply:
Will do.